e2e_test/source_inline/kafka/avro/name_strategy.slt

```diff
@@ -105,7 +105,7 @@ with (
 sleep 2s
 
 
-query II
+query II retry 3 backoff 5s
```

Comment:
If the retries happen frequently, we may increate the `sleep` above instead - does this suggest that the ingestion speed dropped?

Reply:
hmmm another test failed. This is suspicious 🤔

<img width="657" alt="image" src="https://github.com/user-attachments/assets/b522fa54-0fb0-4e1b-9578-3e02c1abbec0" />